### DISCUSSION TITLE
Tamaño de los inputs en testcases

### DISCUSSION THREAD

**Discussion by ManuelV03** [IIC2133-PUC/2024-1#31] [Tarea 0] [answered]
Hola, me surgió una duda sobre si los testcases que usaran para corregir la tarea tendrán tamaños de inputs similares en magnitud a los de los testcases que se nos entregaron o si tendrán algunos valores máximos. Por otro lado, teniendo esto en consideración, ¿es importante preocuparse mucho por la complejidad del algoritmo de ordenamiento que se implemente?

**Comment by sachondo7** [comment, marked as answer]
Sí son parecidos, si con tu código te corren los tests de prueba en menos de 10 segundos deberías andar bien con los tests que se usaran para evaluar. 
Ademas, en unos días podrán probar los tests reales para ver si pasan con el puntaje o no y ahí puedes ver si necesitas optimizar la complejidad de tu código.

**Reply by ManuelV03** [reply]
Gracias!

**Reply by psepulveda99** [reply]
Hola @sachondo7! Cómo se hace eso de poder probar los tests reales que mencionan?

**Reply by sachondo7** [reply]
En los semestres pasados se pasaba un servidor que corre el código que está en tu repo pero por temas del dcc que aun no están resueltos no esta disponible y no tiene pinta que lo vaya a estar antes de la entrega. En todo caso con los testcase que se les pasan les debería bastar para sacarse el 7
